src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -895,6 +895,7 @@ static MethodSymbol getTopLevelMethod(MethodSymbol method)
             foreach (var node in nodes)
             {
                 Visit(node);
+                Unsplit();
```

Comment:
:memo: The following call to `Visit` expects an unsplit state. The scenarios which use this method do not need to preserve the split state anyways.